integration-tests/funqy-google-cloud-functions/README.md

```diff
@@ -0,0 +1,77 @@
+# Google Cloud Functions - Funqy Binding
+
+This integration test has no automated test, it needs to be launched manually.
+
+## Build the artifact
+
+First, you need to log in to Google Cloud:
+
+```shell script
+gcloud auth login
+```
+
+Then you need to use Maven to build the artifact and put it in a directory where it will be the only file.
+
+```shell script
+mkdir deployment
+mvn clean package
+cp target/quarkus-integration-test-funqy-google-cloud-functions-999-SNAPSHOT-runner.jar deployment/
```

Comment:
This copy could be done within the extension post build.  See the lambda extension for an example.

Reply:
OK, I'll create an issue to add this to both Funqy and HTTP Cloud function.